.github/configuration/upgrade-test-configuration.yml

```diff
@@ -149,7 +145,7 @@ upgrade-version: [{
 {
   upgrade-path: [
     {
-      version: 16.2,
+      version: 16.4,
       upgrade-type: null
```

Comment:
Why this change?

Reply:
We already have 16.1 -> latest so 16.2 seems unnecessary, so changed it to a newer version instead.